core/domain/exp_services.py

```diff
@@ -152,6 +174,17 @@ def get_exploration_from_model(exploration_model, run_conversion=True):
 
 
 def get_exploration_summary_from_model(exp_summary_model):
+    """Returns an ExplorationSummary domain object.
+
+    Args: 
+        exp_summary_model: An object of ExplorationSummary Model.
+
+    Returns: 
+        ExplorationSummary domain object containing information like id,
```

Comment:
An ExplorationSummary domain object that represents the given ExplorationSummaryModel.